spring-core/src/main/java/org/springframework/cglib/core/ReflectUtils.java

```diff
@@ -382,7 +382,7 @@ public static Method findNewInstance(Class iface) {
 			}
 		}
 		methods.remove(null);
-		return (Method[]) methods.toArray(new Method[methods.size()]);
+		return (Method[]) methods.toArray(new Method[0]);
```

Comment:
Please revert this change.

Rationale:

_Please refrain from modifying classes under `org.springframework.asm`, `org.springframework.cglib`, and `org.springframework.objenesis`. Those include repackaged forks of the third-party libraries ASM, CGLIB, and Objenesis. Any refactoring to those classes should take place upstream in the originating repository. The Spring Framework will then pick up the changes when syncing with official updates of the forked third-party libraries._

Reply:
OK, thank you for your analysis. @sbrannen